### ISSUE TITLE
Testing new stanify - replicated? datastructure?

### ISSUE THREAD

**Issue by hyunjimoon** [Data4DM/stanify#42]
@Dashadower  below is a vensim file with the datastructure you've requested -- Light blue variable `exog demand` is datastructure and dark blue `table for order fulfillment ratio` is table function. Could you please check below?

1. setting value for `exog demand` makes this variable no longer `datastructure`. If my memory is correct, you were trying to receive data only from vensim. However, I don't know how this can be implemented other than the original method (supplying value in python after declarign datastructure in vensim). Happy to be corrected!

2. time series patterns are replicated in the result from stan's draws2data. For easy comparison, I've set `exog demand` is 2 * Time + 100 (this requires changing `exog demand` type (so it's not datastructure anymore ) as shown below.

![image](https://user-images.githubusercontent.com/30194633/219866967-f7f57ad2-aad7-441b-9108-110bb9758591.png)

With the value above, please make sure overshoot of `adj ss` and smooth adjustment of `adj s` are visible in stan-generated time series.
<img width="1114" alt="image" src="https://user-images.githubusercontent.com/30194633/219866765-4a008862-d1ff-491a-b673-dfdf8aa98e8c.png">

[compare_vensim_stan.mdl.zip](https://github.com/Data4DM/stanify/files/10774282/compare_vensim_stan.mdl.zip)

**Comment by Dashadower** [comment]
In the Vensim model you've provided, I couldn't find any instances of `exog demand` being used with a supplemented argument. Is `exog demand` supposed to be a single scalar value?

**Comment by Dashadower** [comment]
In addition, I request that you provide the Vensim file with `exog demand` changed to an expression. It's hard to modify Vensim files without an actual Vensim client.

**Comment by hyunjimoon** [comment]
>  I couldn't find any instances of exog demand being used 
I don't follow; don't you have the following in the zipped mdl file?
```
EOR= INTEG (
	adjusting eor,
		exog demand)
	~	widget/Week
	~		|

B in=
	exog demand
	~	widget/Week
	~		|
```

**Comment by Dashadower** [comment]
Nah, it's there. It's just that `exog demand` is being used as a static scalar value; I'm not sure when you would generate a function for like like lookup.

**Comment by hyunjimoon** [comment]
From below, exog demand's type is datatype and time series vector is supplied (perhaps through python @ in stanify). in vensim, I recall we implemented as interpolate/lookup structure. Does thie make sense to you? Is everyting good?

exog demand:INTERPOLATE:
	~	widget/Week
	~		|

**Comment by Dashadower** [comment]
Then is the default assumption for Vensim data variables is they exist as a time-vector?

**Comment by hyunjimoon** [comment]
yes
